omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -29,8 +37,11 @@
         // it.birthdateEstimated is of type boolean (doesn't need sanitization)
         birthdateEstimated: ${ it.birthdateEstimated },
         identifier:"${ it.patientIdentifier ? ui.escapeJs(ui.encodeHtmlContent(it.patientIdentifier.identifier)) : '' }"
-    });
-
+    }
+        <% listingAttributeTypeNames.each { attributeName -> %>
+            patientObj["${ attributeName }"] = "${ it.getAttribute(attributeName)?:'' }";
```

Comment:
It should be noted that showing attributes without any JS-escaping or HTML-encoding causes an XSS vulnerability in the patient search widget (I was able to verify this locally). Unfortunately, the fix will not be as simple as calling ui.escapeJs(ui.encodeHtmlContent()) as is done for name and gender fields since it.getAttribute(attributeName) is not of type String.